source/admin_manual/list_of_events.rst

```diff
@@ -14,39 +14,135 @@ See :ref:`event_design` for technical implementation details.
 Categories
 **********
 
-.. TODO: Compile complete list of categories
-
-+----------------+------------------------------------------------------------+
-| Category       | Description                                                |
-+================+============================================================+
-| auth           |                                                            |
-+----------------+------------------------------------------------------------+
-| mailbox        |                                                            |
-+----------------+------------------------------------------------------------+
-| storage        |                                                            |
-+----------------+------------------------------------------------------------+
-| metacache      | Metacache library                                          |
-+----------------+------------------------------------------------------------+
-| sieve          |                                                            |
-+----------------+------------------------------------------------------------+
-| sieve-execute  | Relates to (several) Sieve scripts being                   |
-|                | executed for a particular message.  This envelops all of   |
-|                | Sieve execution; both runtime and action execution.        |
-+----------------+------------------------------------------------------------+
-| sieve-runtime  | Relates to the evaluation of individual scripts.           |
-+----------------+------------------------------------------------------------+
-| sieve-action   | Relates to individual actions executed.                    |
-+----------------+------------------------------------------------------------+
-| sieve-storage  | Sieve storage events                                       |
-+----------------+------------------------------------------------------------+
-| managesieve    | Managesieve action events                                  |
-+----------------+------------------------------------------------------------+
-| smtp-server    | SMTP/LMTP server events                                    |
-+----------------+------------------------------------------------------------+
-| smtp-submit    | SMTP submission client events                              |
-+----------------+------------------------------------------------------------+
-| service:<name> | Event was sent by the named service, e.g. service:imap     |
-+----------------+------------------------------------------------------------+
+Root Categories
+===============
+
++--------------------+---------------------------------------------------------+
+| Category           | Description                                             |
++--------------------+---------------------------------------------------------+
+| auth               | Authentication events.                                  |
```

Comment:
wonder if these should be written as :ref:`auth <name of link below>`

Reply:
The categories can be used by different groups of events though. I think it would be confusing here if it linked to only one of the groups. I was first thinking I'd add list of categories to each event, but then I got tired of thinking about it because it's not always clear what categories exist in which events since it depends a lot on the parents events also.